examples/libf3d/cpp/multi-files/main.cxx

```diff
@@ -33,12 +33,12 @@ int main(int argc, char** argv)
   f3d::window& win = eng.getWindow();
   win.render();
 
-  // Create a timer to stop interaction after one second
+  // Start interaction and stop it after one second
   f3d::interactor& inter = eng.getInteractor();
-  inter.createTimerCallBack(1000, [&inter]() { inter.stop(); });
+  inter.start(1, [&inter]() { inter.stop(); });
```

Comment:
Are these tests executed in the tests? I'm afraid this will confuse readers.

Reply:
yes they are executed and thats why we need this. I've added a comment below showing an actual call but it can explain more.

Reply:
ok fair enough. These examples must be improved at some point anyway.